_topic_maps/_topic_map.yml

```diff
@@ -3262,7 +3262,16 @@ Topics:
 - Name: About MTC
   File: about-mtc
 - Name: MTC release notes
-  File: mtc-release-notes
+  Dir: release_notes
+  Topics:
+  - Name: MTC release notes 1.8
```

Comment:
I see your comment

As I require clear links to the latest release notes, I am not renaming the MTC 1.8 release notes file.
If and when MTC 1.9 is released, this will become mtc-release-notes.adoc and MTC 1.8 release notes will become mtc-release-notes-1-8.adoc.

But why not name `mtc-release-notes-1.8` now? Is the `mtc-release-notes` file if it's only a file and note a branch? Just easier for tracking purposes so one knows the latest release?

Reply:
The [existing published ](https://docs.openshift.com/container-platform/4.15/migration_toolkit_for_containers/mtc-release-notes.html#migration-mtc-release-notes-1-8_mtc-release-notes)notes, state `migration-mtc-release-notes-1-8`.

Reply:
I see what you mean but as there are multiple xrefs to `mtc-release-notes`, doing this way ensures that these links will always point to the latest release notes. 
Your way would mean that the links would have to be updated with each Y release... and it is too easy to forget to update those xrefs and point the reader to out-of-date release notes.

Reply:
Thanks, @anarnold97 . I see where you're coming from. Good to build efficieny into the docs maintenance system as against getting tangled with something tedious like updating numerous xrefs for every release.

Reply:
Yeah and cos my memory ain't what was... and i would forget

Reply:
Yeah because it is the `MTC 1.8 release notes`

So for 1.8 this is correct:

```
- Name: MTC release notes
  Dir: release_notes
  Topics:
  - Name: MTC release notes 1.8
    File: mtc-release-notes
  - Name: MTC release notes 1.7
    File: mtc-release-notes-1-7
 ```
 
 When 1.9 is released we would change to:
 
 ```
 - Name: MTC release notes
  Dir: release_notes
  Topics:
  - Name: MTC release notes 1.9
    File: mtc-release-notes   
// as the .adoc file does not change the xrefs remain pointing to the latest
  - Name: MTC release notes 1.8
    File: mtc-release-notes-1-8
// created by "<save as> as `mtc-release-notes.adoc`"
  - Name: MTC release notes 1.7
    File: mtc-release-notes-1-7    
 ``` 
 
 I could change the name of the file to `mtc-release-notes-latest`, this would make it explicit